diff --git a/igwn_alert/tests/test_igwn_alert.py b/igwn_alert/tests/test_igwn_alert.py
index c3caac6..5acaf8b 100644
--- a/igwn_alert/tests/test_igwn_alert.py
+++ b/igwn_alert/tests/test_igwn_alert.py
@@ -29,7 +29,7 @@ def callback_function(topic=None, payload=None):
 # Test the version number
 def test_version():
     from igwn_alert import __version__
-    assert __version__ == '0.2.0'
+    assert __version__ == '0.2.1'
 
 
 # Test an unauthorized session
